### ISSUE TITLE
TBSCertList version must be 'v2'

### ISSUE THREAD

**Issue by FonsDC** [wbond/crlbuilder#4]
https://github.com/wbond/crlbuilder/blob/6d5977fde1d18ff13ae109cabd29dd94218b670c/crlbuilder/__init__.py#L643

TBSCertList version must be **v2**, as specified in RFC5280:

```
TBSCertList  ::=  SEQUENCE  {
     version                 Version OPTIONAL,
                                   -- if present, MUST be v2
```
Some applications reject a CRL if the version is v3.

**Comment by wbond** [comment]
This does appear to be a bug, thanks for the report!

**Comment by ftbarata** [comment]
Hi @wbond, any updates of this? I'm facing the same problem.

**Comment by ftbarata** [comment]
@FonsDC did you do any workaround of of this? I tried:

```
from asn1crypto.crl import Version
'version': Version(1)
```
Even doing this or removing the  " 'version': Version(1)" from TbsCertList, no success. It keeps 'v3'.

**Comment by wbond** [comment]
This was fixed by #7, and should be part of the next release (date TBD)
